dace/frontend/fortran/ast_desugaring.py

```diff
@@ -0,0 +1,4089 @@
+# Copyright 2019-2025 ETH Zurich and the DaCe authors. All rights reserved.
+
```

Comment:
This file could really use a high-level file doc comment describing the purpose it serves and operations it performs. It's a large and complex file, and the name doesn't make it sufficiently self-explanatory to me. May also be just me, but I'd prefer a comment.